src/core/binding.c

```diff
@@ -151,8 +151,8 @@ QuicBindingInitialize(
         "[bind][%p] Created, Udp=%p LocalAddr=%!SOCKADDR! RemoteAddr=%!SOCKADDR!",
         Binding,
         Binding->DatapathBinding,
-        LOG_BINARY(sizeof(DatapathLocalAddr), &DatapathLocalAddr),
-        LOG_BINARY(sizeof(DatapathRemoteAddr), &DatapathRemoteAddr));
+        CLOG_BYTEARRAY(sizeof(DatapathLocalAddr), &DatapathLocalAddr),
```

Comment:
One benefit of doing something more generic, like `LOG_BINARY` is if we have to pull the plug in CLOG, we don't have to do a global find/replace.